schug/load/biomart.py

```diff
@@ -4,7 +4,7 @@
 import requests
 
 LOG = logging.getLogger(__name__)
-BIOMART_37_URL = "https://feb2014.archive.ensembl.org/biomart/martservice?query="
+BIOMART_37_URL = "https://grch37.ensembl.org/biomart/martservice?query="
 BIOMART_38_URL = "https://www.ensembl.org/biomart/martservice?query="
```

Comment:
Should be the same data, right?

<img width="580" alt="image" src="https://github.com/Clinical-Genomics/schug/assets/28093618/e87b7077-0119-4c91-8258-675c62b5b51a">